projects/admin/src/app/circulation/checkin/checkin.component.ts

```diff
@@ -109,6 +109,9 @@ export class CheckoutComponent implements OnInit {
         this.getPatronInfo(item.loan.patron.barcode);
       }
       if (item.hasRequests) {
+        this.itemsService.doValidateRequest(item, this.loggedUser.library.pid).subscribe(itemData => {
+          item.actionDone = itemData.actionDone;
+        });
```

Comment:
Could you mention this in commit message?

Reply:
This shouldn't be in this PR.